Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/Page.cs

```diff
@@ -56,5 +56,82 @@ public static UIStatusBarAnimation PreferredStatusBarUpdateAnimation(this IPlatf
 			SetPreferredStatusBarUpdateAnimation(config.Element, value);
 			return config;
 		}
+
+		public static readonly BindableProperty UseSafeAreaProperty = BindableProperty.Create(nameof(UseSafeArea), typeof(bool), typeof(Page), false);
+
+		public static bool GetUseSafeArea(BindableObject element)
+		{
+			return (bool)element.GetValue(UseSafeAreaProperty);
+		}
+
+		public static void SetUseSafeArea(BindableObject element, bool value)
+		{
+			element.SetValue(UseSafeAreaProperty, value);
+		}
+
+		public static IPlatformElementConfiguration<iOS, FormsElement> SetUseSafeArea(this IPlatformElementConfiguration<iOS, FormsElement> config, bool value)
+		{
+			SetUseSafeArea(config.Element, value);
+			return config;
+		}
+
+		public static bool UseSafeArea(this IPlatformElementConfiguration<iOS, FormsElement> config)
+		{
+			return GetUseSafeArea(config.Element);
+		}
+
+		public static readonly BindableProperty LargeTitleDisplayProperty = BindableProperty.Create(nameof(LargeTitleDisplay), typeof(LargeTitleDisplayMode), typeof(Page), LargeTitleDisplayMode.Automatic);
+
```

Comment:
is there an example of this being used ?

Reply:
Yes we have gallery now

Reply:
what's the difference between UseLargeTitlesProperty and LargeTitleDisplayProperty ? can't those be combined ?

Reply:
I think we should reflect the iOS api as this is a PlatformSpecific, easy for those o google for info.